### PR TITLE
Create Awesome books application

### PR DESCRIPTION
**Pull request by nachosala89**
- Added index.html structure.
- Displayed book list from Javascript objects.
- Added add section.
- Implemented "remove" and "add" buttons behavior.
- Persist data in localStorage.

### CODE REVIEW

File: js/dom.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+if (localStorage.getItem('books') !== null) {
+  window.addEventListener('load', () => {
+    books = JSON.parse(localStorage.getItem('books'));
+    displayBooksList(books);
+  });
+}
+
+// const removeButtons = document.querySelectorAll();
+// for (let i = 0; i < removeButtons.length; i += 1) {
```
Comment (dasileker) [inline js/dom.js]: - [ ] Please remove the commented codes.

File: js/dom.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  title.textContent = books[index].title;
+  article.appendChild(title);
+  const author = document.createElement('p');
+  author.textContent = books[index].author;
+  article.appendChild(author);
+  const button = document.createElement('button');
+  button.classList.add('remove-btn');
+  button.textContent = 'Remove';
+  button.addEventListener('click', () => { removeBook(books, index); });
+  article.appendChild(button);
```
Comment (dasileker) [inline js/dom.js]: - [ ] Please use the best practice rules in ES6, you should use **arrow function** please.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @nachosala89,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/dom.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+const container = document.querySelector('#books-list');
+
+function createArticle(books, index) {
```
Comment (carlos-ssh) [inline js/dom.js:3]: [OPTIONAL]
- IMPROVE YOUR CODE USING ES6 SYNTAX.

**Review by carlos-ssh — APPROVED** [review]
Hi @nachosala89 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**